rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import commonjs from 'rollup-plugin-commonjs';
+import resolveModule from 'rollup-plugin-node-resolve';
+import typescript from 'rollup-plugin-typescript2';
+import uglify from 'rollup-plugin-uglify';
+import pkg from './package.json';
+
+const GLOBAL_NAME = 'geofire';
+
+const plugins = [
+  resolveModule(),
+  typescript({
+    typescript: require('typescript')
+  }),
+  commonjs()
+];
+
+const external = Object.keys(pkg.dependencies || {});
+
+const completeBuilds = [{
+    input: 'src/index.ts',
+    output: [{
+        file: pkg.browser,
```

Comment:
We are shipping both a `pkg.browser` and a `pkg.main` that are both builds of the same file. Since this package doesn't require a Node/Browser specific entrypoint, let's remove the `pkg.browser` build from here and the `package.json`

_Note: See https://github.com/jshcrowthe/howto-browser-modules for details_